api_specs/v2/component_analyses.yaml

```diff
@@ -8,10 +8,94 @@ servers:
 security:
 - tokenAuth: []
 paths:
+  /component-analyses/:
+    post:
+      tags:
+        - CA batch call
+      summary: Get security and other details about a software component and recommendations to change to a more secured version in a batched manner.
+      parameters:
+        - name: uuid
+          in: header
+          description: User Identification uuid
+          required: false
+          schema:
+            type: string
+      requestBody:
+        description: CA Batch Request
+        required: true
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                ecosystem:
+                  type: string
+                  example: pypi
+                payload:
```

Comment:
Can payload be renamed to packageversions

Reply:
Ack!

Reply:
or dependencies/package_versions/packages?